.pylintrc

```diff
@@ -1,20 +0,0 @@
-[MASTER]
```

Comment:
Pylint still does a fair number of things that ruff cannot (and probably never will). I don't think we should be removing pylint here.

Reply:
Reverted.